src/main/java/ch/uzh/ifi/hase/soprafs22/rest/mapper/DTOMapper.java

```diff
@@ -71,4 +73,7 @@ int convertTeamToTeamNumber(Team team){
     @Mapping(source = "gameMode", target = "gameMode")
     @Mapping(source = "gameMap", target = "gameMap")
     public abstract GameGetDTO convertGameToGameGetDTO(Game game);
+    public Position convertPositionDTOToPosition(PositionDTO position) {
```

Comment:
abstract should be enough, shouldn't it?

Reply:
Indeed, done.

Reply:
Can't do. Position has no setters (and it doesnt need them) so we just build a new one.